portfolio/src/main/webapp/index.html

```diff
@@ -12,7 +12,7 @@
 
   <div id="headercontent">
     <h1>Tommy's Portfolio</h1>
-    <h2>Welcome to my webpage! Click the tabs to learn more about me.</h2>
+    <h2 style="font-size: 30px; color: #e4f5ff;">Welcome to my webpage! Click the tabs to learn more about me.</h2>
```

Comment:
You could alternatively override the style for h2 in the css file. IMO, it would be cleaner and give a consistent styling to all your Heading 2.